viewing_party/utils_02.py

```diff
@@ -0,0 +1,17 @@
+import collections
+
+def get_watched_avg_rating(user_data):
+    watched = user_data['watched']
+    avg_rating = sum(movie['rating'] for movie in watched) / len(watched) if watched else 0
+    return avg_rating
+
+def get_most_watched_genre(user_data):
+    most_watched_genre = None
+    watched = user_data['watched']
+    if watched:
+        genre_count = collections.Counter(movie['genre'] for movie in watched)
```

Comment:
Nice work researching and using collections. As an exercise, on future projects, it would be good practice to build a frequency map yourself: https://www.geeksforgeeks.org/counting-the-frequencies-in-a-list-using-dictionary-in-python/